### ISSUE TITLE
progressIntervalMilliseconds doesn't work for uploadBlob

### ISSUE THREAD

**Issue by ezze** [edeckers/react-native-blob-courier#171]
**Describe the bug**

I am trying to decrease `progressIntervalMilliseconds` from default value `1000` when I `uploadBlob` but it seems that it doesn't make sense which value I specify.

**To Reproduce**

```typescript
const options: BlobUploadInput = {
  url,
  method: 'POST',
  headers: {
    Authorization: `Bearer ${token}`,
    'Content-Type': 'application/octet-stream'
  },
  mimeType: 'application/octet-stream',
  absoluteFilePath: databasePath,
  onProgress: (e: BlobProgressEvent) => {
    console.log((e.written / e.total) * 100)
  },
  progressIntervalMilliseconds: 100
};

const { response } = await BlobCourier.uploadBlob(options);
```

`onProgress` function is called at the same interval (1 second)

**Expected behavior**

`onProgress` function is called each 100 milliseconds

**Smartphone (please complete the following information):**
 - Device: Android Emulator shipped with Android Studio
 - OS: Android 
 - 10 (SDK Platform 29)

**Comment by edeckers** [comment]
Hi @ezze, thank you for your bug report! I just took a quick peek at the code and it looks like you're totally right:
https://github.com/edeckers/react-native-blob-courier/blob/1049b4e0695d4f724e1635595dba7a336ef423b1/android/src/main/java/io/deckers/blob_courier/BlobCourierModule.kt#L43

I expect to fix the problem this week.

**Comment by ezze** [comment]
@edeckers I installed `3.0.0`, and seems it works as expected! I am very grateful to you for the quick fix. :tada: 

By the way, is major release made by mistake? I see that you set `2.0.7` and `2.0.8` as milestones for my issues.

**Comment by edeckers** [comment]
>I installed 3.0.0, and seems it works as expected! I am very grateful to you for the quick fix.

Happy to hear that! And no problem, I really appreciate you taking the time to report this bug, so I can improve the libary :pray: 

>By the way, is major release made by mistake?

I had to bump to a major version because I changed the `minSdkVersion` from `19` to `21`. I don't expect many people to suffer from this change with [SDK 19 being 8 years old](https://developer.android.com/studio/releases/platforms#4.4), but it's a breaking change nonetheless. I will update the names for the Milestones though!
